cs_tools/cli/tools/git/config.py

```diff
@@ -0,0 +1,198 @@
+from typing import Union
+
+import typer
+from httpx import HTTPStatusError
+from rich.align import Align
+from rich.table import Table
+
+from cs_tools.cli.dependencies import thoughtspot
+from cs_tools.cli.types import MultipleChoiceType
+from cs_tools.cli.ux import CSToolsApp, rich_console
+from cs_tools.types import GUID
+
+Identifier = Union[GUID, int, str]
+
+app = CSToolsApp(
+    name="config",
+    help="Tools for working with git configurations.",
+    add_completion=False,
+    invoke_without_command=True,
+    no_args_is_help=True,
+)
+
+
+def _get_token(ctx: typer.Context) -> str:
```

Comment:
@billdback-ts this doesn't get called, what's it supposed to do?

Reply:
Ooops.  That's one I left in when I thought we needed the token.  Removing.